code/__DEFINES/mobs.dm

```diff
@@ -60,9 +60,17 @@
 #define DEFAULT_BODYPART_ICON_ORGANIC 'icons/mob/human_parts_greyscale.dmi'
 #define DEFAULT_BODYPART_ICON_ROBOTIC 'icons/mob/augmentation/augments.dmi'
 
+
 #define MONKEY_BODYPART "monkey"
 #define ALIEN_BODYPART "alien"
 #define LARVA_BODYPART "larva"
+
+
+#define NULL_BODY 0
+#define HUMAN_BODY (1 << 0)
+#define MONKEY_BODY (1 << 1)
+#define ALIEN_BODY (1 << 2)
+#define LARVA_BODY (1 << 3)
```

Comment:
Document these. It's not clear that it's for `allowed_animal_origin`, as well as document the fact they are bitfields.

Reply:
fixed